packages/shipjs/src/step/release/createGitHubRelease.js

```diff
@@ -15,41 +15,44 @@ export default async ({ version, config, dir, dryRun }) =>
       skipIf: cannotUseHub,
     },
     async () => {
-      const { getTagName, releases, updateChangelog } = config;
+      const {
+        getTagName,
+        releases: { assetsToUpload, extractChangelog } = {},
+      } = config;
       const tagName = getTagName({ version });
       const args = [];
 
       // extract matching changelog
-      const changelog = updateChangelog ? getChangelog(version, dir) : null;
+      const getChangelogFn = extractChangelog || getChangelog;
```

Comment:
If we do this, no matter if user uses built-in conventional-changelog or not (`updateChangelog`), user will have a power (`extractChangelog`) to manipulate content for GitHub Release.

When `updateChangelog` is `false`, it will try to use `extractChangelog` and if user hasn't configured it, it will still get changelog from `getChangelog`. At the worst case, if the format of changelog is incompatible, `getChangelog` will simply return `null`. Then the GitHub Release will have an empty content (but still with a correct title).

Does this make sense to you?

Reply:
yes, that's make sense too!